doc/neps/nep-0048-spending-project-funds.rst

```diff
@@ -0,0 +1,421 @@
+.. _NEP48:
+
+=====================================
+NEP 48 — Spending NumPy Project Funds
+=====================================
+
+:Author: Ralf Gommers <ralf.gommers@gmail.com>
+:Author: Inessa Pawson <inessa@albuscode.org>
+:Author: Stefan van der Walt <stefanv@berkeley.edu>
+:Status: Draft
+:Type: Informational
+:Created: 2021-02-07
+:Resolution:
+
+
+Abstract
+--------
+
+The NumPy project has historically never received significant **unrestricted**
+funding. However, that is starting to change.  This NEP aims to provide
+guidance about spending NumPy project unrestricted funds, by formulating a set
+of principles about *what* to pay for and *who* to pay. It will also touch on
+how decisions regarding spending funds get made, how funds get administered,
+and transparency around these topics.
+
+
+Motivation and Scope
+--------------------
+
+In its 16+ year history, the NumPy project has only spent on the order of
+$10,000 USD of funds that were not restricted to a particular program.  Project
+income of this type has been relying on donations from individuals and, from
+May 2019, recurring monthly contributions from Tidelift. By the end of 2020,
+the Tidelift contributions increased to $3,000/month, and there's also a
+potential for an increase of donations and grants going directly to the
+project. Having a clear set of principles around how to use these funds will
+facilitate spending them fairly and effectively. Additionally, it will make it
+easier to solicit donations and other contributions.
+
+A key assumption this NEP makes is that NumPy remains a largely
+volunteer-driven project, and that the project funds are not enough to employ
+maintainers full-time. If funding increases to the point where that assumption
+is no longer true, this NEP should be updated.
+
+In scope for this NEP are:
+
+- Principles of spending project funds: what to pay for, and who to pay.
+- Describing how NumPy's funds get administered.
+- Describing how decisions to spend funds get proposed and made.
+
+Out of scope for this NEP are:
+
+- Making any decisions about spending project funds on a specific project or
+  activity.
+- Principles for spending funds that are intended for NumPy development, but
```

Comment:
I think the definition of un-unrestricted funds are in this sentence but I couldn't fully understand it. Though I feel what is mentioned here.

Reply:
I've already covered it in this comment: https://github.com/numpy/numpy/pull/18454/commits/cf9a5db9bca7af3665c78c071cff49d36214481b#r580023603. 
Non-profit and academic funding is the world of its own.:)

Reply:
If these terms have specific meanings that are not generally understood, they should be defined, maybe as footnotes.

Reply:
@mattip @ilayn What do you think of a short glossary prefacing the document instead of the footnotes?

Reply:
Any supplementary information like a glossary should be at the end, not the beginning.

Reply:
If this is going to be read and not filed away I would expect a definition at the beginning. This is not an article after all so footnote I would really not bother with. It doesn't have to be a definition section anyways. I think the academic style is creeping in here.

Reply:
This is exactly why I had the history section at the start. Basically, unrestricted funds are coming from donations, Tidelift monthly payments, GSoC mentoring fees, and other such things - income for the project that goes into our account and we don't already have a donation for.

This contrasts with grants and other types of contracts, which typically say "you get $xxx,000 for deliverables Y or activities Z".

Reply:
So then if we connect that history to what you just wrote here, I'm sold. I just need to get an understanding of what restricted means such that the rest is easy to parse.

Reply:
> If this is going to be read and not filed away 

If we are successful in our fundraising efforts, I expect this NEP to be used on a regular basis.